samples/bot-all-cards/csharp/BotAllCards/Bots/WelcomeBot.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.BotBuilderSamples
+{
+    // RichCardsBot prompts a user to select a Rich Card and then returns the card
+    // that matches the user's selection.
+    public class WelcomeBot : DialogBot<MainDialog>
```

Comment:
Why this class name is WelcomeBot ?

Reply:
Only the welcome page's details are being loaded. 